package.json

```diff
@@ -86,6 +86,7 @@
     "@typescript-eslint/parser": "^4.14.2",
     "babel-loader": "^8.1.0",
     "babel-eslint": "^10.1.0",
+    "@zkochan/cmd-shim": "^5.1.0",
```

Comment:
Did you add this entry by hand? It's not sorted.

Reply:
I did move its order around. The latest commit should sort everything properly.